Formula/gitoxide.rb

```diff
@@ -0,0 +1,28 @@
+class Gitoxide < Formula
+  desc "Idiomatic, lean, fast & safe pure Rust implementation of Git"
+  homepage "https://github.com/Byron/gitoxide"
+  url "https://github.com/Byron/gitoxide/archive/refs/tags/v0.25.0.tar.gz"
+  sha256 "098bb18e1cae42ab7597b6b442538d3f51b57935a848ea121e20e2921d6a4693"
+  license "Apache-2.0"
+
+  # Cmake is here as only rust install wasn't enough (explained in PR)
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
```

Comment:
```suggestion
  depends_on "rust" => :build
  uses_from_macos "curl"
```
You need this to fix the linkage failure on Linux.